spec/lib/common/json-schema-validator-spec.js

```diff
@@ -92,5 +258,18 @@ describe('JsonSchemaValidator', function () {
             validator.getSchema('/refschema/r2');
         }).to.throw(/can't resolve reference /);
     });
+
+    it('should get schema with reference resolved', function () {
```

Comment:
Suggest adding one more test case:
 - no $ref in the schema, make sure the getSchemaResolved behaves well. Because sometimes the code may work for very complex cases but fail in the very simple case, this case is to make sure it works in both simple & complex cases.